ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/subversion/SubversionCredentialsDialog.java

```diff
@@ -8,33 +8,21 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.plugin.svn.ide.askcredentials;
+package org.eclipse.che.ide.api.subversion;
```

Comment:
Not good for sure but "OK" for now if in case urgent task. @vinokurig please add comment why it here and think in future how to move it to SVN plugin